docs/surveys/developer-survey/README.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Gradle Build Tool - 2024 Software Developer Survey"
+description: >
+  We're doing a developer survey to prioritize our developer experience work.
+  Take the survey and get gift cards and special swag!
+---
+
+# Gradle Build Tool - Software Developer Survey
+
+The Gradle team is committed to enhancing developer experience,
+it is one of the top priorities on [Gradle Roadmap](../..roadmap/README.md/).
+Your feedback is crucial in helping us prioritize our efforts!
+
+<a class="button button--blue"
+   href="https://www.surveymonkey.com/r/7FZPXQZ"
+   target="_blank">Take the Survey
+</a>
+
+## 2024 Developer Survey
+
+![Test](./images/hero.png)
+
+The 2024 survey focuses on Gradle Build Tool developer experience, intends to get feedback on complexity of particular operations and to prioritize them.
+This is why the form is quite long, and we will appreciate your patience.
```

Comment:
❗ I don't think we should say this at all.  Most of our sample respondents completed the original version in 10-15 min, and we think the version you revised should be even faster than that on average.

Reply:
Everything beyond 2 minutes is long. Happy to remove/rework though

Reply:
Regardles of if it _is_ long or not (which I guess depends on how you define long), we shouldn't _advertise_ it as being long, as this might discourage people from responding.